[docs][intro/readme]: add admission simple description (#144)
hwameistor/hwameistor: docs/docs/1.whats_the_hwameistor.md

>   It is used to automatically schedule the Pod to the correct node which has the associated HwameiStor volume. With the scheduler, the Pod doesn't have to has the NodeAffinity or NodeSelector field to select the node. A scheduler will work for both LVM and Disk volumes.

Reviewer: ```suggestion
  It is used to automatically schedule a pod to a correct node which has the associated HwameiStor volumes. With the scheduler, the pod can directly find the correct node without the NodeAffinity or NodeSelector field. A scheduler works for both LVM and Disk volumes.
```

> - **AdmissionController**

Reviewer: ```suggestion
- **admission-controller**
```
